assets/static/querystring.js

```diff
@@ -39,9 +38,20 @@ var QueryString = (function() {
     }
 
 
+    remove = function(key) {
+        var baseUrl = [location.protocol, '//', location.host, location.pathname].join(''),
```

Comment:
Easily tested in a browser's dev tools. Just enter `location.protocol` into the console and you should get `"https:"` back :)